templates/suspenders_layout.html.erb.erb

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8" />
   <meta name="ROBOTS" content="NOODP" />
   <meta name="viewport" content="initial-scale=1" />
+  <!-- change the title in config/locales/en.yml in the en.titles key -->
```

Comment:
I like clarifying the use but am against removing. We *should* be setting
titles per page for a better user experience and for better (and
simple) SEO. It's bad UX to open several tabs to an app and have them all
say "Application Name". Title makes it really simple to fix this once you
know how to use it.

On Saturday, January 24, 2015, Jason Draper <notifications@github.com>
wrote:

> In templates/suspenders_layout.html.erb.erb
> <https://github.com/thoughtbot/suspenders/pull/495#discussion_r23501543>:
>
> > @@ -4,6 +4,7 @@
> >    <meta charset="utf-8" />
> >    <meta name="ROBOTS" content="NOODP" />
> >    <meta name="viewport" content="initial-scale=1" />
> > +  <!-- change the title in config/locales/en.yml in the en.titles key -->
>
> [image: :-1:] I like this gem. It works great for setting custom titles
> depending on your current page.
>
> I don't think that it is fair to say it is confusing, the usage is just a
> bit hidden and pointing it out would be a sufficient way to fix the problem.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/suspenders/pull/495/files#r23501543>.
>